docs/snippets/all/archetypes/transform3d_hierarchy_frames.cpp

```diff
@@ -0,0 +1,101 @@
+// Logs a transform hierarchy using explicit transform frame relationships.
+//
+// ⚠️ This is an experimental feature ⚠️
+
+#include <rerun.hpp>
+
+constexpr float TAU = 6.28318530717958647692528676655900577f;
+
+int main() {
+    const auto rec = rerun::RecordingStream("rerun_example_transform3d_hierarchy_frames");
+    rec.spawn().exit_on_failure();
+
+    rec.set_time_duration_secs("sim_time", 0.0);
+
+    // Planetary motion is typically in the XY plane.
+    rec.log_static("/", rerun::ViewCoordinates::RIGHT_HAND_Z_UP);
+
+    // Setup spheres, all are in the center of their own space:
+    rec.log(
+        "sun",
+        rerun::Ellipsoids3D::from_centers_and_half_sizes({{0.0f, 0.0f, 0.0f}}, {{1.0f, 1.0f, 1.0f}})
+            .with_colors({rerun::Color(255, 200, 10)})
```

Comment:
I think this is overloaded such that

```suggestion
            .with_colors(rerun::Color(255, 200, 10))
```
should also work. But need to consult compiler